packages/react-examples/src/react-tabs/Tabs/Tabs.doc.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
```

Comment:
Let's skip adding this file and the markdown files for now, since they're only used in the legacy demo app and the public site (neither of which really needs to include Tabs yet)